packages/vue/src/components/molecules/SfSteps/SfSteps.stories.js

```diff
@@ -171,12 +171,22 @@ export default {
 const Template = (args, { argTypes }) => ({
   components: { SfSteps },
   props: Object.keys(argTypes),
+  data() {
+    return {
+      currentStepIndex: 0,
+    };
+  },
+  methods: {
+    stepHandler(value) {
+      this.currentStepIndex = value;
```

Comment:
Wdyt about adding `this.change(value);` here to show `change` event in action tab?

Reply:
done